blog/urls.py

```diff
@@ -15,10 +15,16 @@
 """
 from django.conf.urls import url
 from django.contrib import admin
+from django.conf import settings
+from django.conf.urls.static import static
 
 from post import views
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^api/v1/posts/$', views.PostList.as_view())
+    url(r'^api/v1/posts/$', views.PostList.as_view()),
+    url(r'^api/v1/image/$', views.ImageCreate.as_view())
```

Comment:
if view class can only create instances, maybe will be better to change URL to `api/v1/images/create`, and then (if it will be required), use `api/v1/images` to list entities.

Reply:
good idea.